Bottom: 1790151c3c537779e8ea92d5ecb8a01d190bb212
Top:    9ae4117ae645aaa227c437059236b537aa21e207
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-13 14:31:55 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 3b0b0d4..e29d62d 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -331,13 +331,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
